### ISSUE TITLE
Carbon ads on docs.devwithlando.io.

### ISSUE THREAD

**Issue by serundeputy** [thinktandem/tandem#99]
## What is to be done?

We switched docs.devwithlando.io to vuepress and dropped the adsense ads.  

- [x] We need to contact the Carbon folks and get spun up with them.
- [ ] Once we have the carbon creds we need to integrate the ads into Vuepress
  - [ ] Where do we want the ads?
  - [ ] Figure out how to set up `ads.txt` for carbon service: https://github.com/lando/lando/commit/1b52ca4d733dc79af46e5419a8237aab932a0690

## Task checklist

- [x] I'm aware of [what a task actually entails](https://docs.thinktandem.io/guides/improve-tandem.html#suggesting-a-change)
- [x] I've selected _at most_ two labels that _best_ describe what part of the business this ticket improves eg `sales`, `dev`, etc
- [x] I've added a story point estimation. **This should never exceed 3!!!**
- [x] I've created a follow up task or discussion and linked to it in the section below or labeled this as a "one-off" task if there is no follow up task
- [x] I've also gone into any applicable follow up tasks and [set this issue as a dependency](https://help.zenhub.com/support/solutions/articles/43000010349-create-github-issue-dependencies)

## Next steps(s)

Basically a one off, but we'll want to at least loosely monitor that stuff is working and generating some income.

**Comment by pirog** [comment]
@serundeputy i have re-engaged carbon and will forward details along when we have an account set up!

**Comment by pirog** [comment]
@serundeputy and @reynoldsalec pinged you guys with accounts for this!
